### PR TITLE
ES v5.22 docs

### PR DESCRIPTION
**Pull request by esethna**
Includes: Channel sidebar and team drag/drop

### CODE REVIEW

**Comment by esethna** [comment]
@devinbinnie can you help confirm the config values?
image

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -3604,6 +3604,19 @@ Specify the color of the SAML login button text for white labeling purposes. Use
 | This feature's ``config.json`` setting is ``"LoginButtonTextColor": ""`` with string input.                                   |
 +-------------------------------------------------------------------------------------------------------------------------------+
 
+Experimental Sidebar Features (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Disabled: Users cannot access the experimental channel sidebar feature set.
```
Comment (amyblais) [inline source/administration/config-settings.rst:3610]:
```suggestion
**Disabled:** Users cannot access the experimental channel sidebar feature set.  
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3604,6 +3604,19 @@ Specify the color of the SAML login button text for white labeling purposes. Use
 | This feature's ``config.json`` setting is ``"LoginButtonTextColor": ""`` with string input.                                   |
 +-------------------------------------------------------------------------------------------------------------------------------+
 
+Experimental Sidebar Features (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Disabled: Users cannot access the experimental channel sidebar feature set.  
+
+Enabled (Default On): Enables the experimental sidebar features for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. Features include collapsible categories, unread filtering, and history arrows. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_
```
Comment (amyblais) [inline source/administration/config-settings.rst:3612]:
```suggestion
**Enabled (Default On):** Enables the experimental sidebar features for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. Features include collapsible categories, unread filtering, and history arrows. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_ 
```

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -3604,6 +3604,19 @@ Specify the color of the SAML login button text for white labeling purposes. Use
...
 +-------------------------------------------------------------------------------------------------------------------------------+
 
+Experimental Sidebar Features (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Disabled: Users cannot access the experimental channel sidebar feature set.  
+
+Enabled (Default On): Enables the experimental sidebar features for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. Features include collapsible categories, unread filtering, and history arrows. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_ 
+
+Enabled (Default Off): Users must enable the experimental sidebar features in Account Settings. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_
```
Comment (amyblais) [inline source/administration/config-settings.rst:3614]:
```suggestion
**Enabled (Default Off):** Users must enable the experimental sidebar features in Account Settings. `Learn more or give us feedback <https://about.mattermost.com/default-sidebar/>`_.
```

File: source/help/getting-started/organizing-conversations.rst [hunk 4]
```diff
@@ -131,4 +131,25 @@ This will add the channel to a "Favorites" section in the top of the sidebar, so
 
 To remove a channel from the "Favorites" section, select the "Favorite" option again.
 
+--------------------------------------------
+Experimental: Channel Organization Features
+--------------------------------------------
+
+Join us in testing an experimental feature set offering additional functionality for managing channels in your sidebar. The features can be enabled using an opt-in config setting configured by a System Admin: `ExperimentalChannelSidebar <https://docs.mattermost.com/administration/config-settings.html#experimental-channel-sidebar-experimental>`_.
```
Comment (amyblais) [inline source/help/getting-started/organizing-conversations.rst:138]:
```suggestion
Join us in testing an experimental feature set offering additional functionality for managing channels in your sidebar. The features can be enabled using an opt-in config setting configured by a System Admin: `ExperimentalChannelSidebar <https://docs.mattermost.com/administration/config-settings.html#experimental-sidebar-features-experimental>`_. 
```

**Comment by devinbinnie** [comment]
> @devinbinnie can you help confirm the config values?
> image

@esethna Yep, that's correct.

**Comment by amyblais** [comment]
@esethna @devinbinnie Is it not called ``ExperimentalChannelSidebarOrganization``? https://github.com/mattermost/mattermost-server/blob/master/model/config.go#L317

**Comment by devinbinnie** [comment]
> @esethna @devinbinnie Is it not called `ExperimentalChannelSidebarOrganization`? https://github.com/mattermost/mattermost-server/blob/master/model/config.go#L317

Sorry, yes you are correct. I forgot we had it under a different name in the code.

**Comment by esethna** [comment]
@devinbinnie can we please change it to "ExperimentalSidebarFeatures" as per the spec? It was chosen thoughtfully to avoid user error with the existing "ExperimentalChannelOrganization" setting

**Comment by devinbinnie** [comment]
> @devinbinnie can we please change it to "ExperimentalSidebarFeatures" as per the spec? It was chosen thoughtfully to avoid user error with the existing "ExperimentalChannelOrganization" setting

Sure, can you create a ticket for it and assign it to me? It will require code changes on 3 different repos.

**Comment by esethna** [comment]
@devinbinnie https://mattermost.atlassian.net/browse/MM-24059

Thanks for the catch Amy

**Comment by esethna** [comment]
@amyblais can we please merge this tomorrow and get it live on docs? I'd like to get the docs PR merged and live before the blog post for channel sidebar goes out (since the release blog links to it)

**Comment by esethna** [comment]
I updated the config name to ExperimentalChannelSidebarOrganization

It's very confusing with ExperimentalChannelOrganization still existing, but the name change got cut from the release, so updating the docs instead

**Pull request merged**